packages/lisk-p2p/test/unit/peer.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+/* tslint:disable:no-console */
+import { expect } from 'chai';
+
+import { PeerConfig, Peer } from '../../src';
+
+describe('#Peer test', () => {
```

Comment:
`#` referes to method until now, so I think it's good to say `Peer module` or `Peer entity` please check other tests